test/spark/test_spark_chunking.py

```diff
@@ -88,6 +88,38 @@ def test_values_to_keys(sc):
     assert allclose(x, c.values_to_keys((0,)).unchunk().toarray())
     assert allclose(x, c.values_to_keys((0, 1)).unchunk().toarray())
 
+
+def test_padding(sc):
```

Comment:
I might break this into two tests, `test_padding` and `test_padding_errors`, even if it requires duplicating some code. But I generally prefer smaller tests.